### PR TITLE
Mirror libtw2 technical documentation

### PR DESCRIPTION
**Pull request by Zwelf**
If the libtw doc gets updated, running `./update_libtw2_doc.sh` (and committing the change) is sufficient to update the website.

### CODE REVIEW

File: www/_layouts/default.html [hunk 1]
```diff
@@ -48,6 +48,7 @@
             <li><a href="/tournaments/">Tournaments</a></li>
             <li><a href="/skins/">Skin Database</a></li>
             <li><a href="/stats/">Statistics</a></li>
+            <li><a href="/libtw2-doc/">Documentation</a></li>
```
Comment (def-) [inline www/_layouts/default.html:51]: I don't think we want to link it here and in the file below. After all this is not "DDNet Documentation", but "libtw2 Documentation"

Comment (Zwelf) [reply www/_layouts/default.html:51]: Yeah, wasn't sure about this. Do you have a better place to link the doc or should I just not link it for now?

Comment (def-) [reply www/_layouts/default.html:51]: I'm not sure. It seems mostly relevant for developers.

**Comment by Zwelf** [comment]
I've removed the links now. I would be in favor of adding a doc section in general (source code doc, /explain and other stuff), where the libtw2 doc could be a subsection.
I've created an updated version of https://ddnet.tw/explain (with Patiga) in standalone svgs, which I also want to put here soon: https://wiki.ddnet.tw/images/f/f9/Entities_clear.svg (it includes scripts and only works with <embed>, <object> or <iframe> tags)

**Pull request merged**